Restructured  the layout of the current documentation (#312)
notaryproject/notaryproject.dev: content/en/docs/faq.md

title: "Frequently asked questions"

> description: "Frequently asked questions about Notary"

Reviewer: There are multiple other instances on this page, where "Notary" has to either change to "Notation" or "Notary Project"

type: docs
weight: 7
---

> ## What registries are compatible with Notary?

Reviewer: ```suggestion
## What registries are compatible with Notation?
```
Also need to make the same change in the next line "The following registries are compatible with ~~Notary~~ Notation for OCI artifact signing and verification:

## JWS signature envelope

**Q: Why is JWT not used as the signature envelope format?**

**A:** JWT uses JWS compact serialization which do not support unsigned attributes. Notary Project signature requires support for unsigned attributes. Instead we use the *JWS JSON Serialization* representation, which supports unsigned attributes.

**Q: Why JWT `exp` and `iat` claims are not used?**

> **A:** Unlike JWT which always contains a JSON payload, Notation envelope can support payloads other than JSON, like binary. Reusing the JWT payload structure and claims, limits the Notation JWS envelope to only support JSON payload, which is undesirable. Also, reusing JWT claims requires following same claim semantics as defined in JWT specifications. The [`exp`](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4) claim requires that verifier MUST reject the signature if current time equals or is greater than `exp`, where as Notation allows verification policy to define how expiry is handled.

Reviewer 2: ```suggestion
**A:** Unlike JWT which always contains a JSON payload, the Notary Project signature envelope can support payloads other than JSON, like binary. Reusing the JWT payload structure and claims, limits the Notary Project signature JWS envelope to only support JSON payload, which is undesirable. Also, reusing JWT claims requires following the same claim semantics as defined in JWT specifications. The [`exp`](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.4) claim requires that the verifier MUST reject the signature if the current time equals or is greater than `exp`, where as Notation allows verification policy to define how expiry is handled.
```

## Trust store and trust policy

> **Q: Does Notation Specification supports `n` out of `m` signatures verification requirement?**

Reviewer 2: ```suggestion
**Q: Does Notary Project specification supports `n` out of `m` signatures verification requirement?**
```

> **A:** Notation Specification doesn't support `n` out of `m` signature requirement verification scheme.

Reviewer 2: ```suggestion
**A:** Notary Project specification doesn't support `n` out of `m` signature requirement verification scheme.
```

Signature verification workflow succeeds if verification succeeds for at least one signature.

> **Q: Does Notation Specification support overriding of revocation endpoints to support signature verification in disconnected environments?**

Reviewer 2: ```suggestion
**Q: Does Notation support overriding of revocation endpoints to support signature verification in disconnected environments?**
```
